server/src/main/java/org/elasticsearch/index/mapper/GeoShapeFieldMapper.java

```diff
@@ -306,11 +307,12 @@ private SpatialPrefixTree prefixTree() {
     public GeoShapeFieldMapper(String simpleName,
                                int position,
                                long columnOID,
+                               boolean isDropped,
                                @Nullable String defaultExpression,
                                FieldType fieldType,
                                MappedFieldType mappedFieldType,
                                CopyTo copyTo) {
-        super(simpleName, position, columnOID, defaultExpression, fieldType, mappedFieldType, copyTo);
+        super(simpleName, position, columnOID, isDropped, defaultExpression, fieldType, mappedFieldType, copyTo);
     }
 
     @Override
```

Comment:
For Geo shape we need extra parser changes, smth like https://github.com/crate/crate/commit/1ca05298a4efbfeb5df1077bfdaaf126d6382cea#diff-a16d1580140c1a2b00a240081a2d6622d9a906dd3047687e0b61b70d6961b8e5R193-R195

We discussed why we cannot make it rely to generic FieldMapper's parse 0 because of doc values as far as I remember - see [comment](https://github.com/crate/crate/pull/14145#discussion_r1196606726).